src/editor/main.cpp

```diff
@@ -205,16 +205,13 @@ void receive_message_from_webview(const std::wstring& msg) {
 
 void initialize_webview() {
   try {
-    if (!g_webview_process) {
```

Comment:
`g_webview_process` is always null when `initialize_webview()` is called (and it's called only once).